src/MICore/JsonLaunchOptions.cs

```diff
@@ -95,6 +95,12 @@ public abstract partial class BaseOptions
         /// </summary>
         [JsonProperty("setupCommands", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public List<SetupCommand> SetupCommands { get; protected set; }
+
+        /// <summary>
+        /// Optional parameter. If true, the debugger should stop after connecting to the target.
+        /// </summary>
+        [JsonProperty("stopAtConnect", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public bool? StopAtConnect { get; set; }
```

Comment:
We should add this to the openFolder schema: src/MIDebugPackage/OpenFolderSchema.json

Reply:
[Added](https://github.com/microsoft/MIEngine/pull/1109/files#diff-98a456ea0533a77610e519a56b3866fac24085519f4d42522a09ffbb4fa593ebR176).